README.md

```diff
@@ -1,19 +1,19 @@
 # bagit-firecloud-lambda
 
-This lambda accepts a POST with a bagit in the request body, and uploads TSVs from the bagit to FireCloud. When
+This lambda accepts a POST with a BagIt in the request body, and uploads TSVs from the bagit to FireCloud. When
 FireCloud adds an endpoint that directly accepts a bagit, this lambda will be unnecessary.
 
-A development instance is currently stood up at `https://egyjdjlme2.execute-api.us-west-2.amazonaws.com`.
+A development instance is currently stood up at `https://vzltaytzg4.execute-api.us-west-2.amazonaws.com/api/`.
 
 ## Using the service
 
 ```
-curl -X POST --header "Authorization: Bearer <Google token>" -header "Content-type: application/octet-stream" --data @bagit.zip "https://egyjdjlme2.execute-api.us-west-2.amazonaws.com/api/exportBag?workspace=<workspace>&namespace=<namespace>" 
+curl -X POST --header "Authorization: Bearer <Google token>" -header "Content-type: application/octet-stream" --data @bagit.zip "https://vzltaytzg4.execute-api.us-west-2.amazonaws.com/api/exportBag?workspace=<workspace>&namespace=<namespace>" 
 ```
 
 * Google token -- a Google oauth token. This is used when making the FireCloud API calls. The FireCloud API currently
 requires that this token have the `https://www.googleapis.com/auth/devstorage.full_control` scope.
-* bagit.zip -- A Bagit, zipped up. The Bagit currently must contain one TSV file, although it will be changed to 
+* bagit.zip -- A BagIt , zipped up. The BagIt  currently must contain one TSV file, although it will be changed to 
```

Comment:
This is no longer true; it now expects two files, doesn't it?

Reply:
true